scripts/exrap_dump.sh

```diff
@@ -761,7 +762,7 @@ else
 
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_5} 1 aircft \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_5} 1 aircft \
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 aircft \

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_5} 1 aircft \